addons/arsenal/stringtable.xml

```diff
@@ -155,115 +155,138 @@
         </Key>
         <Key ID="STR_ACE_Arsenal_loadoutDeleted">
             <English>The following loadout was deleted:</English>
             <French>L'équipement suivant fut supprimé:</French>
+            <German>Folgende Ausrüstung wurde entfernt:</German>
             <Polish>Następujący zestaw został skasowany:</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_loadoutUnshared">
             <English>The following loadout is not public anymore:</English>
             <French>L'équipement suivant n'est plus public:</French>
+            <German>Folgende Ausrüstung ist nicht mehr öffentlich:</German>
             <Polish>Następujący zestaw nie jest już publiczny:</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_saveEmptyNameBox">
             <English>The name field is empty!</English>
             <French>Le champ nom est vide !</French>
+            <German>Das Feld "Name" ist leer!</German>
             <Polish>Pole nazwy jest puste!</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_saveAuthorError">
             <English>You are the author of this loadout</English>
             <French>Vous êtes l'auteur de cet équipement</French>
+            <German>Du bist der Ersteller dieser Ausrüstung</German>
             <Polish>Jesteś autorem tego zestawu</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_saveSharedError">
             <English>A loadout of yours with the same name is public</English>
             <French>Un de vos équipements avec le même nom est public</French>
+            <German>Eine deiner Ausrüstungen mit dem gleichen Namen ist öffentlich</German>
             <Polish>Jeden z Twoich zestawów nazwany tak samo jest już publiczny</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_loadoutSaved">
             <English>The following loadout was saved:</English>
             <French>L'équipement suivant fut enregistré:</French>
+            <German>Folgende Ausrüstung wurde gespeichert:</German>
             <Polish>Następujący zestaw został zapisany:</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_loadoutLoaded">
             <English>The following loadout was loaded:</English>
             <French>L'équipement suivant fut chargé:</French>
+            <German>Folgene Ausrüstung wurde geladen:</German>
             <Polish>Następujący zestaw został wczytany:</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_renameExistError">
             <English>A loadout with the same name already exist!</English>
             <French>Un équipement avec le même nom existe déjà !</French>
+            <German>Eine Ausrüstung mit dem gleichen Namen existiert bereits!</German>
             <Polish>Zestaw z tą nazwą już istnieje!</Polish>
         </Key>
         <Key ID="STR_ACE_Arsenal_loadoutRenamed">
             <English>was renamed to</English>
             <French>fut renommé en</French>
+            <German>wurde umbenannt in</German>
```

Comment:
`wurde umbenannt zu` ?
`Das Loadout wurde umbenannt zu "Loadout"`
`Das Loadout wurde umbenannt in "Loadout"`

Reply:
https://www.duden.de/rechtschreibung/umbenennen

`in`